components/UntrackedRateMixedChart.vue

```diff
@@ -340,14 +350,14 @@ const options: ThisTypedComponentOptionsWithRecordProps<
               ].toLocaleString()
               let label = `${
                 this.dataLabels[tooltipItem.datasetIndex!]
-              } : ${cases} ${unit}`
+              } : ${cases} 人`
```

Comment:
こちら、`人`が翻訳されず、固定表示になってしまいますので、
```suggestion
              } : ${cases} ${this.$t('人')}`
```
とした方がよいと思います。

Reply:
こちらも修正いたします。